cli/cmd/vuln_host_scan_package_manifest.go

```diff
@@ -245,7 +244,7 @@ func filterHostScanPackagesVulnDetails(vulns []api.VulnerabilitySoftwarePackage)
 	out := make([]api.VulnerabilitySoftwarePackage, 0)
 
 	for _, vuln := range vulns {
-		if vulCmdState.Fixable && !vuln.HasFix() {
+		if vulCmdState.Fixable && !vuln.HasFix() || !vuln.IsVulnerable() {
```

Comment:
This is wrong. This will only work when the user passes the `--fixable` flag.

We need this in a different if statement.

Reply:
Added tests and fixed this